examples/sha256_ivc.rs

```diff
@@ -0,0 +1,233 @@
+use std::marker::PhantomData;
```

Comment:
Please add a [module-level doc](https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#module-level-vs-type-level-docs) indicating what the purpose of this module is and how it differs from the neighboring `sha256.rs` example.

Reply:
Done